docs/book/src/clusterctl/developers.md

```diff
@@ -98,6 +98,12 @@ please note that each `provider_repo` should have its own `clusterctl-settings.j
 
 The Docker provider is not designed for production use and is intended for development environments only.
 
+[kind] is not designed for production use.
+
+**Minimum [kind] supported version**: v0.7.0
+
+**If you are using v0.8.0+ you must export the variable `KIND_EXPERIMENTAL_DOCKER_NETWORK`**
```

Comment:
For sake of simplicity, I would drop this note given that we are making this point down in the doc

Reply:
+1

Reply:
note dropped